src/gui/tagfiltermodel.cpp

```diff
@@ -105,8 +105,6 @@ TagFilterModel::TagFilterModel(QObject *parent)
 
 TagFilterModel::~TagFilterModel()
```

Comment:
It is needed as a consequence of your request to store TagModelItem as objects instead of pointers. The alternative is moving TagModelItem's declaration to the header rather than forward-declaring it.